### PR TITLE
Near sdk collections docs update

### PR DESCRIPTION
**Pull request by deveusss**
Updated docs for Vector, TreeMap, LookupSet, LookupMap, UnorderedMap, UnorderedSet

### CODE REVIEW

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:167]:
```suggestion
```

Don't really need this for this doc test

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:182]:
```suggestion
    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m");
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:195]:
```suggestion
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:215]:
```suggestion
    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m");
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:224]:
```suggestion
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:236]:
```suggestion
    ///
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:240]:
```suggestion
    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m");
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:221]: optional: perhaps change this to iterating from 30 to show off the exclusive nature of the start index?

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:249]:
```suggestion
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:265]:
```suggestion
    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m");
```

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:267]: You're missing a closing brace here. Wondering why the need for two vectors if just indicating which keys to insert

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:273]: you probably need a `}` here, the loop above is not closed

File: near-sdk/src/collections/tree_map.rs
Comment (austinabell) [inline near-sdk/src/collections/tree_map.rs:278]:
```suggestion
```

File: near-sdk/src/collections/unordered_map.rs
Comment (austinabell) [inline near-sdk/src/collections/unordered_map.rs:50]:
```suggestion
    /// use near_sdk::collections::LookupSet;
    /// let mut set: LookupSet<u32> = LookupSet::new(b"m");
```

File: near-sdk/src/collections/unordered_set.rs
Comment (austinabell) [inline near-sdk/src/collections/unordered_set.rs:35]:
```suggestion
    /// use near_sdk::collections::UnorderedSet;
    /// let mut set: UnorderedSet<u32> = UnorderedSet::new(b"m");
```

File: near-sdk/src/collections/vector.rs
Comment (austinabell) [inline near-sdk/src/collections/vector.rs:47]:
```suggestion
    /// use near_sdk::collections::Vector;
    /// let mut set: Vector<u32> = Vector::new(b"m");
```

**Review by austinabell — CHANGES_REQUESTED** [review]

File: near-sdk/src/collections/lookup_map.rs
Comment (thedeveus) [reply near-sdk/src/collections/lookup_map.rs:37]: sure, I will update to byte version)

File: near-sdk/src/collections/lookup_set.rs
Comment (ChaoticTempest) [inline near-sdk/src/collections/lookup_set.rs:28]:
```suggestion
    /// let mut set: LookupSet<u32> = LookupSet::new(b"m");
```

File: near-sdk/src/store/lookup_map/mod.rs
Comment (ChaoticTempest) [inline near-sdk/src/store/lookup_map/mod.rs:306]:
```suggestion
    /// if let Some(x) = map.get_mut(&1) {
    ///     *x = "b";
    ///     assert_eq!(map[&1], "b");
    /// }
```

**Review by ChaoticTempest — COMMENTED** [review]
`near-sdk/.DS_Store` and `near-sdk/src/.DS_Store` seems unnecessary to be adding to the repo. Best to remove those before merging this PR. Aside from that, just some minor stuff still left, but I'll leave it to @austinabell to approve and merge

**Comment by austinabell** [comment]
It seems you are still making additions. Can you let us know when you are done so we can review once, or consider making new additions in a new PR to break it up?

**Comment by thedeveus** [comment]
> It seems you are still making additions. Can you let us know when you are done so we can review once, or consider making new additions in a new PR to break it up?
> 
> Thanks so much for adding all of these examples!

I think i finished with store collections, I am not sure if I need to add more examples in current collections

**Comment by austinabell** [comment]
Not all doc tests are passing, if you run `cargo test` you should be able to see which ones to fix :)

**Comment by deveusss** [comment]
> Not all doc tests are passing, if you run `cargo test` you should be able to see which ones to fix :)

fixed, thank you:)

**Comment by mikedotexe** [comment]
Thanks @deveusss for doing this. Looks like we have a merge conflict to clean up

**Comment by austinabell** [comment]
Is there anything I can do to help resolve conflicts and/or fix the CI? Seems like this PR has gone a bit stale

**Comment by deveusss** [comment]
> 
yes, there is just two different version of examples and need to choose one) I ll fix this

**Comment by mikedotexe** [comment]
Just updated the master and CI is running. we'd love to see this get in, and I'll ask @deveusss if everything is ready to go? I see there's a change requested currently

**Pull request merged**
